functions/supplier/sftp-supplier/src/test/java/org/springframework/cloud/fn/supplier/sftp/SftpSupplierApplicationTests.java

```diff
@@ -399,8 +444,8 @@ private void deleteNewSource(Path newSource) throws IOException {
 	}
 
 	@SpringBootApplication
+	@Import(SpelExpressionConverterConfiguration.class)
```

Comment:
As I said in the comment for the POM change, this must go onto the `SftpSupplierConfiguration`.
The point is even without your change we still have a problem on the `SftpSupplierProperties.Factory.knownHostsExpression` property.
So, your contribution via this PR is really great!

Reply:
`SpelExpressionConverterConfiguration` should be imported automatically by auto-configuration, so the fact it is in the POM should be enough. This also happens in other modules (cfr. sftp-supplier, elasticsearch-consumer, wavefront-consumer, and analytics-consumer) that explicitly import `SpelExpressionConverterConfiguration` only in test classes as the auto-configuration does not apply to them.

Reply:
OK. I missed the fact that it is present as an auto-configuration entry.
Why then it doesn't work in the test since I see the `@SpringBootApplication`?

Reply:
It does actually work. Removed.